scripts/autodoc

```diff
@@ -80,6 +80,13 @@ rm -rf ./autodoc/output
 ./autodoc/upgrading/generate.lua && \
 ./autodoc/pdk/generate.lua
 
+exit_code=$?
```

Comment:
Wonder if it'd be worth it to just turn on `errexit`/`set -e` for this script and then play a couple rounds of "CI failure whack-a-mole" to catch any other lurking problems? Could also turn on `xtrace`/`set -x` for help w/ debugging in the mean time.

Reply:
My personal experience with trying to add `set -e` / [bash strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/) after-the-fact is that a script has to be written from the ground up to support it. Otherwise subshells and redirects and a couple other loose ends don't carry that initial assumption of how to fail.

So I'd opt for this solution.